ome_zarr/writer.py

```diff
@@ -160,7 +145,7 @@ def write_multiscales_metadata(
     group: zarr.Group,
     paths: List[str],
     fmt: Format = CurrentFormat(),
-    axes: List[str] = None,
+    axes: Union[str, List[str], List[Dict[str, str]]] = None,
```

Comment:
https://github.com/ome/omero-cli-zarr/pull/93/files#diff-8ebd9fed3089ce55b4b0d98840330514c910c84da182036847677d2edb02ed54R333 also introduces `transformations` as an extra keyword argument to the equivalent method in `omero-cli-zarr`. Is that something we should cater for? Or not applicable to this PR?

Reply:
It's not "axes" specific, but I think it still makes sense to include in this PR since it's all "v0.4"

Reply:
Added in 5607786